### DISCUSSION TITLE
Display errors thrown by <SignIn /> & <SignUp />

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [clerk/javascript#2033] [Support request - DO NOT OPEN HERE]
Hi, 

I'm building a custom auth pages with Clerk NextJS SDK, but I'm having trouble handling errors with it. 

For example, if I try to log in to an account that's not registered yet, the form throws a 422 error, but I have no idea how to catch it and display it in a toast or something else. 

![CleanShot 2023-11-03 at 11 30 17](https://github.com/clerk/javascript/assets/358412/2c1dea7d-7649-467c-9287-d46eb1e0b5a5)

Any ideas?

**Comment by soumyadeep2002ss** [comment]
I'm facing the same issue when working with custom authentication pages using Clerk in Next.js.

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
